src/styles/main.scss

```diff
@@ -12,3 +12,21 @@ body {
 
 @import "_header.scss";
 @import "_catalog.scss";
+
+@media screen and (min-width: 488px) {
+  .catalog {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media screen and (min-width: 768px) {
+  .catalog {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media screen and (min-width: 1024px) {
+  .catalog {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
```

Comment:
move it to catalog.scss and use scss nesting

Reply:
This is not fixed. Mentor expected this code to be nested. Like this:
```
.catalog {
  ....
  @media screen and (min-width: 1024px) {
      grid-template-columns: repeat(4, 200px);
  }
}
```